site/static/main.css

```diff
@@ -277,10 +278,31 @@ nav a:hover {
   flex: 0 0 auto;
 }
 
+#edit-this-page {
+  display: inline-block;
+  font-size: 0.9em;
+  color: #aaa;
+  padding-top: 30px;
+  cursor: pointer;
```

Comment:
This is weird. It feel like I can click on the whole text but it's not possible.